packages/frontity-org-theme/src/processors/background-color.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { css } from "frontity";
+import { Processor } from "@frontity/html2react/types";
+import FrontityOrg from "../../types";
+
+const colorClassRegex = /has-(\w+)-background-color/;
+
+const backgroundColor: Processor<React.HTMLProps<HTMLElement>, FrontityOrg> = {
+  name: "backgroundColor",
+  test: ({ node }) => {
+    return (
+      node.type === "element" &&
+      node.props.className &&
+      node.props.className.split(/ +/).includes("has-background")
```

Comment:
@DAreRodz did you find a class that was separated by more than one space or did you use that regexp because it's safer than `.split(" ")`?

Reply:
I did it just for "safety", but I think it would have worked the same, two or more spaces would have generated empty strings in between, so...

Do you think `.split(" ")` is more clear? I can change this.

Reply:
Hmm... maybe it's a bit clearer for people not used to work with regexp, yeah. And you're right, I didn't notice that more than one space will return an empty item, which won't affect the result :)